tests/suites/os/tests/config-json/index.js

```diff
@@ -14,6 +14,51 @@
 
 'use strict';
 
+const setConfig = async (test, context, key, value) => {
+
+	return context.systemd.waitForServiceState(
+			'config-json.service',
+			'inactive',
+			context.link
+	).then(() => {
+		if (!value) {
+			test.comment(`Removing ${key} from config.json...`);
+			return context.worker.executeCommandInHostOS(
+				[
+					`tmp=$(mktemp)`,
+					`&&`, `jq`, `"del(.${key})"`, `/mnt/boot/config.json`,
+					`>`, `$tmp`, `&&`, `mv`, `"$tmp"`, `/mnt/boot/config.json`
+				].join(' '),
+				context.link);
+		} else {
+			if (typeof(value) == 'string') {
+				value = `"${value}"`
+			} else {
+				value = JSON.stringify(value);
+			}
+
+			test.comment(`Setting ${key} to ${value} in config.json...`);
+			return context.worker.executeCommandInHostOS(
+				[
+					`tmp=$(mktemp)`,
+					`&&`, `jq`, `'.${key}=${value}'`, `/mnt/boot/config.json`,
+					`>`, `$tmp`, `&&`, `mv`, `"$tmp"`, `/mnt/boot/config.json`
+				].join(' '),
+				context.link);
+		}
```

Comment:
Wondering whether this construct is common enough to abstract it as part of the framework:
* Check for a service to be active
* Wait until the command is executed
* Wait for service to be active again